src/OFS/Image.py

```diff
@@ -520,6 +520,9 @@ def _read_data(self, file):
         n = 1 << 16
 
         if isinstance(file, str):
+            raise ValueError("Must be bytes")
```

Comment:
We should really use six here. When I did the first Python 3 changes on Zope2 itself, I didn't use six. Those were done when I simply ran flake8/pep8 over the codebase since none of the stack was compatible yet. Since we use six in almost all packages incl. Zope2, we should just convert my initial custom `sys.version_info` checks.